frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -159,6 +139,28 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
                     )}
                 </div>
             </div>
+            <div className="mt-3 ml-11">
+                {addon.features?.length > 3 && (
```

Comment:
I wonder if we do like:

- Filter out features that have the same name as the product
- Then if it's greater than 1, show it

Fine with what you have here though if you want to keep it.

Reply:
Yeah, this is a very rudimentary way to do it. That is a good start though. I can add that.